test/src/main/java/org/jvnet/hudson/test/HudsonTestCase.java

```diff
@@ -501,7 +506,19 @@ protected Hudson newHudson() throws Exception {
         File home = homeLoader.allocate();
         for (Runner r : recipes)
             r.decorateHome(this,home);
-        return new Hudson(home, createWebServer(), useLocalPluginManager ? null : pluginManager);
+        if (useLocalPluginManager) {
+            pluginManager = new LocalPluginManager(home) {
+                @Override
+                protected Collection<String> loadBundledPlugins() {
+                    // Overriding so we can force loading of the detached plugins for testing
```

Comment:
I did it because one of the tests (that turns on `useLocalPluginManager`) failed and I'm saying there may be other tests outside Jenkins core that may also be using this (but I don't know).

Tbh, this doesn't seem like something worth spending time debating.